docs/source/algorithms/LoadNexus-v1.rst

```diff
@@ -12,9 +12,25 @@ Description
 The algorithm LoadNexus will read the given Nexus file and try to
 identify its type so that it can be read into a workspace. The file name
 can be an absolute or relative path and should have the extension .nxs
-or .nx5. Currently only Nexus Muon Version 1 files are recognised, but
-this will be extended as other types are supported such as
-:ref:`algm-LoadNexusProcessed`.
+or .nx5.
+
+The type of Nexus file is identified as follows:
+
+* If the file has a group of class ``SDS`` of name ``"definition"`` or ``"analysis"`` 
+  with value ``"muonTD"`` or ``"pulsedTD"``, 
+  then it is taken to be a muon Nexus file and :ref:`algm-LoadMuonNexus` is called.
+
+* Else if main entry is ``"mantid_workspace_1"``
+  then it is taken to be a processed Nexus file and :ref:`algm-LoadNexusProcessed` is called.
+  *The spectrum properties are ignored in this case.*
+
+* Else if main entry is ``"raw_data_1"``
+  then it is taken to be an ISIS Nexus file and :ref:`algm-LoadISISNexus` is called.
+
+* Else if instrument group has a ``"SNSdetector_calibration_id"`` item, 
+  then :ref:`algm-LoadTOFRawNexus` is called.
```

Comment:
Then I'm confused by this line `Mantid::NeXus::NXEntry entry = root.openEntry(root.groups().front().nxname);`.  What is the effect of this line then?

Unrelated to this PR, I noticed that the docs test give an example of every type of file loaded by `LoadNexus`, except for this "NSdetector_calibration_id". Could we find a small example of this type of file and add it?